src/resolvers/articleResolvers.ts

```diff
@@ -44,5 +50,34 @@ export const resolvers = {
         article.supportedLanguages[0];
       return fetchCoreElements(article.grepCodes, language, context);
     },
+    async crossSubjectTopics(
+      article: GQLArticle,
+      args: { subjectId: string; filterIds: string },
+      _: any,
+      context: Context,
+    ): Promise<GQLTopic[]> {
+      const crossSubjectCodes = article.grepCodes.filter(code =>
+        code.startsWith('TT'),
+      );
+      const language =
+        article.supportedLanguages.find(lang => lang === context.language) ||
+        article.supportedLanguages[0];
+      const crossSubjectTopicInfo = await fetchCrossSubjectTopicsByCode(
+        crossSubjectCodes,
+        language,
+        context,
+      );
+      const topics = await fetchSubjectTopics(
+        args.subjectId,
+        args.filterIds,
+        context,
+      );
+      return crossSubjectTopicInfo.map(
+        compGoal =>
+          topics.find(
+            (topic: { name: string }) => topic.name === compGoal.title,
+          ) || { name: compGoal.title, id: '' },
+      );
```

Comment:
Ville egentlig ikke gjøre det sånn, men hvis tverrfaglige temaer i det gitte miljøet ikke har et emne med tilsvarende tittel som grepcodesene så vil jeg sende med tittelen og så kan den heller linke tilbake til tverrfaglige temaer-siden. Kunne kanskje laget et eget interface for akkurat disse tverrfaglige temaene, men det er jo litt unødvendig det også...